### PR TITLE
Replace dangerjs with probot-changelog

### PR DESCRIPTION
**Pull request by bjankord**
### Summary
Update to latest major version

### CODE REVIEW

**Review by StephenEsser — APPROVED** [review]

**Review by emilyrohrbough — APPROVED** [review]

**Review by yuderekyu — APPROVED** [review]

**Comment by emilyrohrbough** [comment]
Actually, can we remove danger and use the github changelog bot like we've started to do in other repos?

**Comment by bjankord** [comment]
@emilyrohrbough Removed danger.js and replaced with probot-changelog integration in this commit: https://github.com/cerner/terra-toolkit/pull/308/commits/0d86ce23a6d9fd9ffb79f1aaa8ea64a278cc3088

File: .travis.yml [hunk 1]
```diff
@@ -36,9 +36,9 @@ jobs:
         - >
           mkdir -p $HOME/docker && docker images -a --filter='dangling=false' --format '{{.Repository}}:{{.Tag}} {{.ID}}'
           | xargs -n 2 -t sh -c 'test -e $HOME/docker/$1.tar.gz || docker save $0 | gzip -2 > $HOME/docker/$1.tar.gz'
-    - stage: lint jest and danger
+    - stage: lint jest
       script:
-        - "travis_wait docker-compose run -e DANGER_GITHUB_API_TOKEN=$DANGER_GITHUB_API_TOKEN -e HAS_JOSH_K_SEAL_OF_APPROVAL=$HAS_JOSH_K_SEAL_OF_APPROVAL -e TRAVIS_PULL_REQUEST=$TRAVIS_PULL_REQUEST -e TRAVIS_REPO_SLUG=$TRAVIS_REPO_SLUG test-ci bash -c 'npm run danger && npm run lint && npm run jest'"
+        - "travis_wait docker-compose run -e HAS_JOSH_K_SEAL_OF_APPROVAL=$HAS_JOSH_K_SEAL_OF_APPROVAL -e TRAVIS_PULL_REQUEST=$TRAVIS_PULL_REQUEST -e TRAVIS_REPO_SLUG=$TRAVIS_REPO_SLUG test-ci bash -c 'npm run lint && npm run jest'"
```
Comment (emilyrohrbough) [inline .travis.yml:41]: I could be wrong, but i think we can drop all of these envs we pass to this container.

Comment (emilyrohrbough) [reply .travis.yml:41]: @ryanthemanuel

Comment (bjankord) [reply .travis.yml:41]: My preference would be to handle that in a separate PR.

Comment (emilyrohrbough) [reply .travis.yml:41]: These appear to be specific to danger: https://github.com/danger/danger/blob/master/lib/danger/ci_source/travis.rb#L30

Comment (bjankord) [reply .travis.yml:41]: [`HAS_JOSH_K_SEAL_OF_APPROVAL` is a Travis CI default environment variable](https://docs.travis-ci.com/user/environment-variables/#default-environment-variables), not necessarily a danger variable.

**Review by mjhenkes — APPROVED** [review]

**Pull request merged**